docs/tutorials/mongoid-documents.txt

```diff
@@ -176,6 +177,51 @@ Types that are not supported as dynamic attributes since they cannot be cast are
 - ``DateTime``
 - ``Range``
 
+The StringifiedSymbol Type
+--------------------------
+
+When using the built-in ``Symbol`` field type, Mongoid defaults to serializing the field into the ``BSON::Symbol``
+type. For more information on the ``BSON::Symbol`` type, see
+`here <https://docs.mongodb.com/mongoid/master/tutorials/mongoid-documents/#bson-symbol-type>`_.
+``BSON::Symbol`` is deprecated, so the ``StringifiedSymbol`` type allows the use of the built-in ``Symbol``
+type while ensuring interoperability with other drivers. The ``StringifiedSymbol`` type stores all data
```

Comment:
I would make this sentence a bit more direct by saying something like, "Use the ``StringifiedSymbol`` field type in order to use the built-in ``Symbol`` type in an application while ensuring..." If there is a best practice, we should tell the user this explicitly.